vscode-dotnet-runtime-library/src/Acquisition/WinMacGlobalInstaller.ts

```diff
@@ -235,16 +235,23 @@ This report should be made at https://github.com/dotnet/vscode-dotnet-runtime/is
     private async downloadInstaller(installerUrl: string): Promise<string>
     {
         const ourInstallerDownloadFolder = IGlobalInstaller.getDownloadedInstallFilesFolder(installerUrl);
-        await this.file.wipeDirectory(ourInstallerDownloadFolder, this.acquisitionContext.eventStream);
         const installerPath = path.join(ourInstallerDownloadFolder, `${installerUrl.split('/').slice(-1)}`);
 
+        if (await this.file.exists(installerPath) && await this.installerFileHasValidIntegrity(installerPath))
```

Comment:
Is it always the same installer?

Reply:
No, it will be different depending upon the version / architecture